src/migrations/20230621141239-refactor-api-token-permissions.js

```diff
@@ -0,0 +1,28 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+    DELETE FROM permissions WHERE permission IN ('CREATE_API_TOKEN', 'UPDATE_API_TOKEN', 'DELETE_API_TOKEN', 'READ_API_TOKEN');
+    DELETE FROM permissions WHERE permission = 'UPDATE_ROLE';
+    DELETE FROM permissions WHERE permission IN ('CREATE_ADMIN_API_TOKEN', 'UPDATE_ADMIN_API_TOKEN', 'DELETE_ADMIN_API_TOKEN', 'READ_ADMIN_API_TOKEN');
```

Comment:
I would not delete these permissions just yet to prevent potential backward compatibility problems. Note that the old version will still be running while these permissions are deleted

Reply:
Reminder that `ADMIN` overrides this anyways and custom root roles are not available for customers yet, but if we feel safer only adding a migration with these deletions after we make sure the code changes are there, we can do that.